angular/gulpfile.js

```diff
@@ -0,0 +1,8 @@
+ var gulp = require('gulp');
+ var sass = require('gulp-sass');
+ 
+ gulp.task('sass', function() {
+	gulp.src('src/scss/*.scss')	 
+		.pipe(sass())
+		.pipe(gulp.dest('src/css'));
+ });
```

Comment:
Good work! Put this in `feature/angular/yak-shaving` instead... And check that it works. ;)